.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+## Summary (required)
+
+- Resolves #issue_number
+
+_Include a summary of proposed changes._
+
+### Required reviewers
+_Include who is required to review prior to merge._
+
+_For example: One designer and two front end developer reviews are required prior to merge._
+
+## Impacted areas of the application
+List general components of the application that this PR will affect:
+
+-  
+
+## Screenshots
+- _Include a screenshot of the new/updated features in context (“in the wild”). If it is an interface change, include both before and after screenshots._
+
+## Related PRs
+List related PRs against other branches:
+
+branch | PR
+------ | ------
+fix/other_pr | [link]()
+feature/other_pr | [link]()
+
+## How to test
+Include any information that may be helpful to the reviewer(s).
+_This might include:_
+
+- links to sample pages to test
+- Any local environmental setup that is unusual (env vars, API version to point to, etc)
+
+## System architecture updates (if applicable)
+_If this pull request changes our [current system diagram](https://github.com/fecgov/FEC/wiki/2.-FEC-system-diagram), include a description of those changes here and update the system diagram._
```

Comment:
Should there be some kind of approval queue for updating the diagram? If one were to make a change and update the system diagram, but then change isn't accepted, the diagram has already been changed.

Reply:
I added an extra step to create a new ticket to update the diagram. But it's not contingent on merging the work.